std/socket.d

```diff
@@ -85,10 +85,10 @@ else version(Posix)
         }
     }
 
+    public import core.sys.posix.netinet.in_;
```

Comment:
Argh - this is needed because the Windows branch is `public import core.sys.windows.winsock2;`